src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -156,17 +156,22 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         Result previousBuildResult = null;
         AbstractBuild<?, ?> previousBuild = build.getPreviousCompletedBuild();
 
-        if (previousBuild != null) {
-            previousBuildResult = previousBuild.getResult();
+        if (forceNewIssue) {
+            currentBuildResultFailure(build, listener, null, filename, vars);
         }
-
-        if (currentBuildResult != Result.ABORTED && previousBuild != null) {
-            if (currentBuildResult == Result.FAILURE) {
-                currentBuildResultFailure(build, listener, previousBuildResult, filename, vars);
+        else {
+            if (previousBuild != null) {
+                previousBuildResult = previousBuild.getResult();
             }
 
-            if (currentBuildResult == Result.SUCCESS && !newIssue) {
-                currentBuildResultSuccess(build, listener, previousBuildResult, filename, vars);
+            if (currentBuildResult != Result.ABORTED && previousBuild != null) {
+                if (currentBuildResult == Result.FAILURE) {
+                    currentBuildResultFailure(build, listener, previousBuildResult, filename, vars);
+                }
+
+                if (currentBuildResult == Result.SUCCESS && !forceNewIssue) {
```

Comment:
You can remove the additional reference to `forceNewIssue` here, since it isn't possible to enter this block if `forceNewIssue` is true.